smoke-test/test_e2e.py

```diff
@@ -1381,3 +1381,115 @@ def test_native_user_endpoints(auth_session):
     )
     remove_user_response.raise_for_status()
     assert "errors" not in remove_user_response
+
+
+def test_system_info_main_endpoint(auth_session):
+    """Test that main system info endpoint returns expected structure with authentication."""
+    response = auth_session.get(f"{auth_session.gms_url()}/openapi/v1/system-info")
+    response.raise_for_status()
+    system_info = response.json()
+    
+    # Verify main response structure
+    assert "springComponents" in system_info, "Missing springComponents in system info response"
+    assert "properties" in system_info, "Missing properties in system info response"
+    
+    logger.info("Main system info endpoint test passed")
```

Comment:
I added these end to end tests to check the configuration is being returned. I think these are more useful than unit tests for those simple classes. However smoke testing takes forever for some reason